docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -63,41 +63,40 @@ init(const char *name)
 
 ## Showing and Hiding Quickpanel Window
 
-To show and hide quickpanel window when your application's window is activated, call the following functions:
+To show or hide quickpanel window when your application's window is activated, call the following functions:
 
 ```
-   ...
-
    if (show)
      tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
    else
      tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
-
-   ...
 ```
 
 ## Getting Visible Status of Quickpanel Window
-To know the state of current visibility of qp service's window, call the `tzsh_quickpanel_visible_get` function. The quickpanel service's window may be visible or not visible on the screen.
+To know the state of current visibility of quickpanel service's window, call the `tzsh_quickpanel_visible_get` function. The quickpanel service's window may be visible or invisible on the screen.
 
 ```
-   ...
    tzsh_quickpanel_state_visible_e state;
 
    tzsh_quickpanel_visible_get(tzsh_qp, &state);
    if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN)
-     /* visible state */
-     ;
+     {
+        /* visible state */
+     }
    else if (state == TZSH_QUICKPANEL_STATE_VISIBLE_HIDDEN)
-     /* invisible state */
-     ;
+     {
+        /* invisible state */
+     }
    else
-     /* error */
-     ;
+     {
+        /* error */
+     }
+
 ```
 
 
 ## Registering Changed Event for the Quickpanel Window
```

Comment:
## Register a Changed Event for Quickpanel Window

Reply:
ok.